notes-packaging/src/main/assemblies/notes-addon-package.xml

```diff
@@ -42,24 +42,8 @@
       <includes>
         <include>${project.groupId}:*:jar</include>
         <include>commons-configuration:commons-configuration:jar</include>
```

Comment:
![image](https://user-images.githubusercontent.com/594824/170483542-0f3edcf8-0149-4479-bec9-67212433a854.png)
apparently this dependency isn't included, same as for (net.sf.json-lib) https://github.com/Meeds-io/notes/pull/540/files#diff-b97cf76091695f7a02119d50c025ed92c53613d4368e8d86502825052dae5899R46